lib/SearchAndSort/SearchAndSort.js

```diff
@@ -90,6 +90,7 @@ class SearchAndSort extends React.Component {
     columnWidths: PropTypes.object,
     createRecordPath: PropTypes.string,
     customPaneSub: PropTypes.node,
+    customPaneSubText: PropTypes.string,
```

Comment:
Are you 100% confident this should be `string`? To accept a `<FormattedMessage>` I would have expected `node`, or perhaps `oneOfType([PropTypes.string, PropTypes.element])`.

Reply:
I add oneOfType.

Reply:
You are right, It's better to use node to accept a `<FormattedMessage>`. So, I change a logic a little bit and now we can use node.